src/main.rs

```diff
@@ -66,6 +73,26 @@ fn main() {
     let opts = Opts::from_args();
     info!("Mains() `opts` arguments are {:?}", opts);
 
+    let config = dirs::config_dir();
+
+    let mut config_path = match config {
+        Some(x) => x,
+        None => panic!("Couldn't find config dir."),
+    };
+    config_path.push("rustscan");
+    config_path.push("config.toml");
+
+    if opts.appdirs {
+        // prints config file location and exits
+        println!("The config file is expected to be at {:?}", config_path);
+        exit(1);
+    }
+
+    let ip = match opts.ip {
```

Comment:
Since why are failing without the IP, why do we need to make it optional?

Reply:
The `appdirs` argument is for the config file, Specifically, it will point the user to the location of the config file on their system. When a user runs this flag, they are expecting to see the file path of the config file and not scan at the same time.

If we do not make the IP address optional, the user cannot run the `appdirs` argument unless they also scan at the same time :) 

My personal pet peeve with programs & config files is that they never show you _where_ to put the file, as it is different per system. This flag will help the user solve that little problem :)

Reply:
Interesting! I think it's a good idea to mention the config directory. :) 
Could we output this path during the program execution and hide it if the user pass the `quiet` flag? This way we wouldn't need one more option for arguments.

This way we achieve:

1. IP is always required
2. The user is informed of the config directory unless they explicitly pass the `quiet` flag.
3. The interface (command arguments) stays the same for now.

Reply:
We can even put this inside the `print_opening()` function, which might become the banner + extra info we want to pass to our users place. 😄 What do you think?

Reply:
This sounds good! We could write something like:
> "Checking for config file at {location}"
Let me make this into a GitHub issue :)